pyuvdata/uvdata/uvdata.py

```diff
@@ -8991,6 +9035,7 @@ def upsample_in_time(
 
     def downsample_in_time(
         self,
+        *,
         min_int_time=None,
```

Comment:
Ok. It isn't actually required for this method because you can either pass `min_int_time` or `n_times_to_avg` so that was the reason I put the asterisk before it. But I can move it if you think it makes more sense.

Reply:
After talking about this function so more we decided to keep it as is.